1.format_data/format_training_data.py

```diff
@@ -0,0 +1,190 @@
+    print(f"Processed cell at: {plate}/{well}/{frame}, location: {cell_x_y}")
+    if cell_segmentation_data.empty:
+        print("No segmentation data match found for this cell!")
+    else:
+        single_cell_data = single_cell_data.to_frame().transpose()
+        single_cell_data.insert(
+            0,
+            "Mitocheck_Object_ID",
+            cell_segmentation_data["Mitocheck_Object_ID"].item(),
+        )
+        cell_phenotypic_class = get_cell_class(
+            single_cell_data, trainingset_file_url, plate, well, frame
+        )
+        if cell_phenotypic_class == None:
+            print("This cell was not found in trainingset.dat!")
+        single_cell_data.insert(0, "Mitocheck_Phenotypic_Class", cell_phenotypic_class)
+
+    return single_cell_data
+
+
+def format_training_data(
+    mitocheck_data_version_url: str, save_path: pathlib.Path, compression: str
+) -> pd.DataFrame:
+    """Add Mitocheck_Object_ID and Mitocheck_Phenotypic_Class fields to each single cell and compile all the cells into a single training data dataframe
+
+    Args:
+        mitocheck_data_version_url (str): url with path to desired version of raw mitocheck_data
+        save_path (pathlib.Path): path to save training data
+        compression (str): type of compression to use when saving dataframe
+
+    Returns:
+        pd.DataFrame: completed training data with Mitocheck_Object_ID and Mitocheck_Phenotypic_Class for each cell
+    """
+    trainingset_file_url = (
+        f"{mitocheck_data_version_url}/0.download_data/trainingset.dat"
+    )
+    segmentation_data_dir = f"{mitocheck_data_version_url}/2.segment_nuclei/segmented/"
+    preprocessed_features_url = f"{mitocheck_data_version_url}/4.preprocess_features/data/normalized_training_data.csv.gz"
+
+    preprocessed_features = pd.read_csv(preprocessed_features_url, compression="gzip")
+    print("Loaded preprocessed features!")
+
+    training_data = []
+    for index, row in preprocessed_features.iterrows():
+        single_cell = row
+        completed_single_cell = complete_single_cell(
+            single_cell, trainingset_file_url, segmentation_data_dir
+        )
+        training_data.append(completed_single_cell)
+
+    training_data = pd.concat(training_data)
+    training_data.to_csv(save_path, compression=compression)
+    return training_data
+
+
+# ### Format training data
+
+# In[3]:
+
+
+base_url = "https://raw.github.com/WayScience/mitocheck_data/"
+# hash changes depending on desired version of mitocheck_data being used
+hash = "de21b9c3201ba4298db2b1704f3ae510a5dc47e2"
+mitocheck_data_version_url = f"{base_url}/{hash}"
+
+output_dir = pathlib.Path("data/")
+output_dir.mkdir(parents=True, exist_ok=True)
+save_path = pathlib.Path(f"{output_dir}/training_data.csv.gz")
+compression = "gzip"
+
+training_data = format_training_data(mitocheck_data_version_url, save_path, compression)
+
+
+# In[4]:
+
+
+print(training_data.shape)
+
+
+# In[5]:
+
+
+print(training_data.head())
```

Comment:
```suggestion
print(training_data.shape)
training_data.head()
```

This saves print space, reduces notebook execution by 1, and prints the .head() nicely

Reply:
(when you make this change, don't actually accept this suggestion. Instead, change it in the notebook and then re-nbconvert)

Reply:
Done in https://github.com/WayScience/phenotypic_profiling_model/pull/1/commits/888fe264d9cc3f1c5d5231ae6f1282cef0dc897d